msg/templates/uorb_microcdr/microRTPS_client.cpp.em

```diff
@@ -86,54 +88,41 @@ void* send(void* /*unused*/)
     uint16_t header_length = 0;
 
     /* subscribe to topics */
-    int fds[@(len(send_topics))] = {};
-
-    // orb_set_interval statblish an update interval period in milliseconds.
 @[for idx, topic in enumerate(send_topics)]@
-    fds[@(idx)] = orb_subscribe(ORB_ID(@(topic)));
-    orb_set_interval(fds[@(idx)], _options.update_time_ms);
```

Comment:
This is not used anymore. Please remove the CLI option or make sure it is used.

Reply:
Yes forgot that one thanks.